body.go

```diff
@@ -1,12 +1,32 @@
 package postman
 
+//These constants represent the available raw languages.
+const (
+	HTML       string = "html"
+	Javascript string = "javascript"
+	JSON       string = "json"
+	Text       string = "text"
+	XML        string = "xml"
+)
+
 // Body represents the data usually contained in the request body.
 type Body struct {
-	Mode       string      `json:"mode"`
-	Raw        string      `json:"raw,omitempty"`
-	URLEncoded interface{} `json:"urlencoded,omitempty"`
-	FormData   interface{} `json:"formdata,omitempty"`
-	File       interface{} `json:"file,omitempty"`
-	GraphQL    interface{} `json:"graphql,omitempty"`
-	Disabled   bool        `json:"disabled,omitempty"`
+	Mode       string       `json:"mode"`
+	Raw        string       `json:"raw,omitempty"`
+	URLEncoded interface{}  `json:"urlencoded,omitempty"`
+	FormData   interface{}  `json:"formdata,omitempty"`
+	File       interface{}  `json:"file,omitempty"`
+	GraphQL    interface{}  `json:"graphql,omitempty"`
+	Disabled   bool         `json:"disabled,omitempty"`
+	Options    *BodyOptions `json:"options,omitempty"`
+}
+
+//BodyOptions holds body options.
```

Comment:
Same here, please add a space after `//` :
```suggestion
// BodyOptions holds body options.
```